### PR TITLE
(maint) Remove Ruby 1.9.3 and 2.0.0 from Travis test matrix

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,17 +4,8 @@ sudo: false
 notifications:
   email: false
 rvm:
-  - 1.9.3
-  - 2.0.0
   - 2.1.1
```
Comment (underscorgan) [inline .travis.yml]: Can we add some more recent rubies while we're in here? maybe a 2.3 and a 2.4?

Comment (mwaggett) [inline .travis.yml]: sure! Do we want to check rubocop on everything? Before we were just checking the latest version.

Comment (underscorgan) [inline .travis.yml]: no, one rubocop is enough, those won't vary based on ruby version.

Comment (mwaggett) [inline .travis.yml]: @underscorgan updated!

**Pull request merged**